### ISSUE TITLE
Arch Linux installation instructions do not work properly

### ISSUE THREAD

**Issue by fsmiamoto** [erpalma/throttled#289]
The current README instructions for Installation on Arch Linux using the Community package do not work properly
since the systemd service file is still called `lenovo_fix`.

```sh
$ pacman -Ql throttled | grep service
throttled /usr/lib/systemd/system/lenovo_fix.service
```

I think the proper fix would be to update the package but maybe we can temporarily update the README just to avoid confusion:
```
$ pacman -S throttled
$ sudo systemctl enable --now lenovo_fix.service
```

Let me know what you think and thanks for the awesome work!

**Comment by iuliandita** [comment]
Config file, too, is still `/etc/lenovo_fix.conf`.

**Comment by avalonv** [comment]
Came here confused about this

**Comment by yangniao23** [comment]
You might try installing throttled-git from the AUR.

**Comment by fsmiamoto** [comment]
Yup, that would work but my point is about having misleading docs on the README

**Comment by CSZongzi** [comment]
This issue should have been fixed after throttled version 0.10.0-1. PR may no longer needed.
